learning/prompts/documentation-lookup/15_advanced_xlang.md

```diff
@@ -0,0 +1,15 @@
+Prompt:
+What is a multi-language pipeline in Apache Beam?
+Response:
+Beam lets you combine transforms written in any supported SDK language (currently, [Java](https://beam.apache.org/documentation/programming-guide/#1311-creating-cross-language-java-transforms) and [Python](https://beam.apache.org/documentation/programming-guide/#1312-creating-cross-language-python-transforms)) and use them in one multi-language pipeline. For example, a pipeline that reads from a Python source, processes the data using a Java transform, and writes the data to a Python sink is a multi-language pipeline.
```

Comment:
Beam lets you combine transforms written in any supported SDK language (currently, [Java](https://beam.apache.org/documentation/programming-guide/#1311-creating-cross-language-java-transforms) and [Python](https://beam.apache.org/documentation/programming-guide/#1312-creating-cross-language-python-transforms)) and use them in one multi-language pipeline. For example, a pipeline that reads from a Python source processes the data using a Java transform, and writes the data to a Python sink is a multi-language pipeline.

Reply:
Done